plugins/analysis-icu/build.gradle

```diff
@@ -42,3 +42,17 @@ tasks.named("yamlRestCompatTest").configure {
     'analysis_icu/10_basic/Normalization with deprecated unicodeSetFilter'
   ].join(',')
 }
+
+tasks.named('splitPackagesAudit').configure {
+    // Lucene packages should be owned by Lucene!
```

Comment:
What do these packages have to do with Lucene?

Reply:
This was a copy/paste error. I'll fix the comments.